### ISSUE TITLE
Setting `TMPDIR` to a location in `migratefs` breaks bash completion

### ISSUE THREAD

**Issue by kcgthb** [stanford-rc/fuse-migratefs#12]
Bash completion uses `read` in a yet-to-be-determined way, that doesn't play well when `TMPDIR` is set to some location on `migratefs`

Demonstration:
```
$ export TMPDIR=/tmp
$ while read -r tmp; do echo $tmp; done <<< "foo"
foo

$ export TMPDIR=$SCRATCH
$ while read -r tmp; do echo $tmp; done <<< "foo"
-bash: read: read error: 0: No such file or directory
```

**Comment by kcgthb** [comment]
https://srcc.uservoice.com/admin/tickets/3035

**Comment by thiell** [comment]
fixed subconsciously in 0.4.x
